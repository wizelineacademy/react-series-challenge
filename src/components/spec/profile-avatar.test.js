import React from 'react';
import renderer from 'react-test-renderer';

import { ProfileAvatar } from '../';

describe('<ProfileAvatar />', () => {
  test('renders data correctly', () => {
    const tree = renderer.create(<ProfileAvatar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
