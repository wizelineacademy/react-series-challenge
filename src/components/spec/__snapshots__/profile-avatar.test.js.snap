// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileAvatar /> renders data correctly 1`] = `
<img
  className="sc-htoDjs iOyEEs"
/>
`;
